providers/implementations/macs/cmac_prov.c

```diff
@@ -213,11 +271,11 @@ static int cmac_set_ctx_params(void *vmacctx, const OSSL_PARAM params[])
 #ifdef FIPS_MODULE
         {
             const EVP_CIPHER *cipher = ossl_prov_cipher_cipher(&macctx->cipher);
-            int approved = EVP_CIPHER_is_a(cipher, "AES-256-CBC")
-                           || EVP_CIPHER_is_a(cipher, "AES-192-CBC")
-                           || EVP_CIPHER_is_a(cipher, "AES-128-CBC");
 
-            if (!approved) {
+            if (!EVP_CIPHER_is_a(cipher, "AES-256-CBC")
+                    && !EVP_CIPHER_is_a(cipher, "AES-192-CBC")
+                    && !EVP_CIPHER_is_a(cipher, "AES-128-CBC")
+                    && !EVP_CIPHER_is_a(cipher, "DES-EDE3-CBC")) {
```

Comment:
Note that I cant do the TDES check here because of ordering issues (see notes in commit), so all it does is allow it here to start with.